client/src/components/frontend/ResourceList/Filters.js

```diff
@@ -1,17 +1,62 @@
 import React, { Component, PropTypes } from 'react';
-import { Link } from 'react-router';
 import capitalize from 'lodash/capitalize';
+import get from 'lodash/get';
 
 export default class ResourceListFilters extends Component {
 
   static displayName = "ResourceList.Filters";
 
   static propTypes = {
-    kinds: PropTypes.array
+    kinds: PropTypes.array,
+    tags: PropTypes.array,
+    filterChangeHandler: PropTypes.func.isRequired
   };
 
-  constructor() {
```

Comment:
You've changed this component so that it needs `kinds` and `tags` props to function properly, but you haven't gone and made the change to the parent that uses this component to ensure that it still functions. Take a look at `components/frontend/ResourceCollection/Detail`. That component includes `ResourceList.Filters`, but doesn't pass it tags — so when you drill into a collection on a project, you end up with the map in `ResourceList.Filters` render method throwing an exception.

Think about how you can make the filters component more flexible and error-resistant. If it's not passed tags or kinds, shouldn't it just not render those parts of the filters? Or, if it's going to always render them, then at least set default props on it:

```
static defaultProps = {
  kind: [],
  tags: []
};
```